cromwell_tools/tests/test_cromwell_tools.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+import io
+import requests
+import requests_mock
+import unittest
+try:
+    # if python3
+    import unittest.mock as mock
+except ImportError:
+    # if python2
+    import mock
+
+from cromwell_tools import cromwell_tools
+
+
+class TestUtils(unittest.TestCase):
+
+    @requests_mock.mock()
+    def test_start_workflow(self, mock_request):
+        """Unit test using mocks
+        """
+        wdl_file = io.BytesIO(b"wdl_file")
```

Comment:
If I'm interpreting this correctly, I think using BytesIO in this way could be misleading for someone who uses this package -- we don't want to imply that the user should pass bytes containing the file name for any of these parameters -- they need to be passing bytes data, right?. If so, I would change these names from `b"wdl_file"` to `b"wdl_file_data"` for each of the inputs.

Reply:
Yes, good call, changing